Makefile.frankfurt

```diff
@@ -18,6 +18,13 @@ INSTALL_DIRECTORY := .venv
 # default configuration
 ENV_FILE ?= .env.default
 
+define load_env
+    $(eval ENV := $(1))
+    @echo " - load env $(ENV)"
+    $(eval include $(1))
+    $(eval export)
+endef
```

Comment:
I'm not an expert in makefile and bash but what are the advantage and disadvantage of this technique in comparison to a more simple way as in #3926 

```bash
environ:
   export $(shell cat $(ENV_FILE)) && \
   ...
```
With your `load_env` does the variable stays after the make ?

Reply:
thats simpler, i will test it, thank you.

Reply:
@ltshb i have used your approach, less code and simpler, thank you!